### ISSUE TITLE
Question - specific trainign scenario

### ISSUE THREAD

**Issue by Westerby** [ErgastiAlex/LDM-Diffusion-sem#4]
Dear authors,

thank you very much for publishing your code.

I've been conducting a research about controllable face editing, especially via use of additional masks as a guiding cue and I came across your paper. I wonder if there's a possibility to alter your architecture in a way that the training data is as follows:

- input1: image of a person A
- input 2: corresponding mask of an image of a person A (eg. masked mouth, eyes, or hair)
- target 1: another image of a person A, but with changed area for a input 2 mask.

In literature I found quite a lot of papers using "reference" or "style" image of another person - than person we want to edit - for face editing, but didn't come across this specific scenario: controllable, mask-guided edit, with paired image in the training process.

Maybe you found such research in your work?

Best

**Comment by ErgastiAlex** [comment]
Hi,
I don't remember any specific paper related to the mentioned topic. However, it is certainly possible to adapt our work in such way. You only have to change how the loss is implemented, by calculating it on the target 2. However, I don't think it is ideal to do so in a Diffusion Model

**Comment by Westerby** [comment]
Thank you very much for answering. As to part of your answer - diffusion model not being ideal - what kind of model would be better in your opinion? A guided GAN?

**Comment by ErgastiAlex** [comment]
The fact is that the power of DM is that you actually don't need any target1, but the model can learn in a self-supervised way how to produce that target, as in our paper. You can try to adapt the DM loss or switch to a guided GAN.

**Comment by Westerby** [comment]
Thanks a lot!
